pkgs/games/openxray/default.nix

```diff
@@ -49,20 +52,30 @@ stdenv.mkDerivation rec {
   cmakeBuildType = "RelWithDebInfo";
   dontStrip = true;
 
+  makeWrapperArgs = lib.optionals stdenv.hostPlatform.isLinux [
+    # Needed because of dlopen module loading code
```

Comment:
Can't we patch the code instead?

Reply:
I think just setting `dontPatchELF = true` so it doesn't strip away `$out/lib` from the RPATHs prolly works better, will do that instead.

Reply:
We could prolly patch the package prefix into [the module loading code](https://github.com/OpenXRay/xray-16/blob/d9e484c7ebcb35321ad3a59b999b86b83628a0db/src/xrCore/ModuleLookup.cpp#L25-L63) but that seems needlessly limiting to me, and it's more effort than just making sure we don't trample over [upstream's rpath-setting code](https://github.com/OpenXRay/xray-16/blob/d9e484c7ebcb35321ad3a59b999b86b83628a0db/CMakeLists.txt#L39C1-L45) by not stripping away what they're setting.